static/panes/compiler.ts

```diff
@@ -695,6 +707,18 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
             insertPoint.addChild(createOptView());
         });
 
+        this.container.layoutManager
+            .createDragSource(this.stackUsageButton, createStackUsageView() as any)
```

Comment:
`createStackUsageView` should be without the `()` here

Reply:
That's a good catch, I'm surprised it still worked with the ()